src/main/java/frc/robot/RobotContainer.java

```diff
@@ -257,6 +266,15 @@ public class RobotContainer {
   /** Singleton instance of {@link ZeroGyro} for the whole robot. */
   public static ZeroGyro zeroGyro = new ZeroGyro();
 
+  /** Singleton instance of {@link DisabledLight} for the whole robot. */
+  public static DisabledLight disabledLights = new DisabledLight();
+
+  /** Singleton instance of {@link EnableLights} for the whole robot. */
+  public static EnabledLight enabledLights = new EnabledLight();
+
+  /** Singleton instance of {@link NodeLight} for the whole robot. */
+  public static NodeLight nodeLight = new NodeLight();
```

Comment:
The gamepieces this year are so super definitely called "notes" and not "nodes."